connectors/src/connectors/github/lib/hierarchy.ts

```diff
@@ -42,9 +42,9 @@ async function getGithubCodeDirectoryParentIds(
       directory.parentInternalId,
       repoId
     );
-    return [directory.parentInternalId, ...parents];
+    return [directory.internalId, directory.parentInternalId, ...parents];
   } else if (directory.parentInternalId === `github-code-${repoId}`) {
-    return [`github-code-${repoId}`, `${repoId}`];
+    return [directory.internalId, `github-code-${repoId}`, `${repoId}`];
```

Comment:
We don't need this as the id is already added in retrieveContentNodeParents . This code is actually not used  by activities for upserting/updatings doc.